web/src/app/core/services/repository.service.ts

```diff
@@ -51,4 +51,27 @@ export class RepositoryService {
     const callable: any = this.fns.httpsCallable('deleteGitWebhookRepository');
     return callable({ repositoryUid, token: this.authService.profile.oauth.githubToken });
   }
+
+  public getRepoRating(repo: RepositoryModel): number {
+    let rating: number = 0;
+    const issuePoints: number = repo.issues.length > 0 ? this.getPoints(repo.issues[0].createdOn) : 0;
+    const releasesPoints: number = repo.releases.length > 0 ? this.getPoints(repo.releases[0].createdOn) : 0;
+    rating = (issuePoints + releasesPoints) * 5;
+    return rating;
+  }
+
+  public getPoints(dateStr: Date): number {
```

Comment:
`dateStr: Date`, the variable name implies `string` but the type states `Date`. Better to just use the type and not have it as part of the variable name

Reply:
sure !!